### PR TITLE
RF: Use the new Streamlines API for orienting of streamlines.

### PR DESCRIPTION
**Pull request by arokem**

### CODE REVIEW

**Comment by skoudoro** [comment]
can you rebase this one @arokem?

**Comment by arokem** [comment]
Rebased! Thanks for reminding me.

File: dipy/tracking/streamline.py [hunk 1]
```diff
@@ -484,7 +484,9 @@ def _orient_list(out, roi1, roi2):
         min1 = np.argmin(dist1, 0)
         min2 = np.argmin(dist2, 0)
         if min1[0] > min2[0]:
-            out[idx] = sl[::-1]
+            out[idx][:, 0] = sl[::-1][:, 0]
```
[the remark this answers is not in the document]
Comment (arokem) [reply dipy/tracking/streamline.py]: streamline objects don't let you assign in place

**Comment by Garyfallidis** [comment]
Okay, thanks.

**Pull request merged**